core/PrevPage.go

```diff
@@ -25,16 +25,8 @@ func PrevPage() {
 		if types.Page == 2 && i != 2 {
 			clear[runtime.GOOS]()
 			var questions []input.Questions
-			questions = append(questions, input.Questions{
-				"Which value are you changing?",
-				32,
-				false,
-			})
-			questions = append(questions, input.Questions{
-				"What is the new value?",
-				32,
-				false,
-			})
+			questions = append(questions, input.NewQuestions("Which value are you changing?", 32, false))
```

Comment:
When initializing a structure, it is better to explicitly state field names to ensure that in case of changes in the order of these fields or the names of the fields, they will correctly continue to be addressed. I prefer ctor functions; it is more readable (biased).